helm/cert-exporter/templates/daemonset.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       labels:
         app: cert-exporter
       annotations:
-        releasetime: {{ $.Release.Time }}
+        releaseRevision: {{ $.Release.Revision | quote }}
```

Comment:
Does `$.Release.Revision` support helm v2? if not, can we check $.Release.Time is nil or not first? 
I know we are going to use it for helm3, but so many times we are deploying it without knowing which helm version for it.

Reply:
Okay I checked it, it supports 
https://v2.helm.sh/docs/chart_template_guide/#built-in-objects

Reply:
@tomahawk28 Thanks for checking.

Yes they actually removed Release.Revision from Helm 3 but then added it again. https://github.com/helm/helm/issues/6690

We just want to roll pods so revision also works.